homework_2/program.js

```diff
@@ -0,0 +1,22 @@
+var ArgStart;
+var Test;
+var ArgSum;
+var ArgNumMax;
+var ArgNumCount;
+
+ArgStart = 2; // ignore first two arguments
```

Comment:
It might be cleaner to define these where you declare them.
eg: var ArgStart = 2;